LibreNMS/Enum/AlertScheduleBehavior.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * AlertScheduleBehavior.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       https://www.librenms.org
+ */
+
+namespace LibreNMS\Enum;
+
+class AlertScheduleBehavior
```

Comment:
This should be enum type, not a class.

Reply:
Ah okay, then the pseudo enum classes are just legacy? Well, I'll try out the actually enum stuff then.